packages/terra-responsive-element/src/ResponsiveElement.jsx

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React, { PropTypes } from 'react';
+import ResizeObserver from 'resize-observer-polyfill';
+import classNames from 'classnames';
+import getBreakpoints from './breakpoints';
+import './ResponsiveElement.scss';
+
+const propTypes = {
+  /**
+   * An element to be displayed at default breakpoints ( Between 0 and tiny )
+  */
+  defaultElement: PropTypes.element,
```

Comment:
I went back and forth on this. This component could be used to hide content at breakpoints smaller than a specific size. If we decide this component forces content to be displayed we can make this required.